schema.d.ts

```diff
@@ -247,6 +247,21 @@ export interface Attribute {
    */
   defaultValue?: string;
 
+  /**
+   * Possible values of the attribute, if any.
+   */
+  possibleValues?: string[];
+  
+  /**
+   * If Type is numeric and has a minimum constraint.
+   */
+  min?: string;
+  
+  /**
+   * If Type is numeric and has a maximum constraint.
+   */
+  max?: string;
+  
```

Comment:
In that case that you had these values by default, would they not likely be defined by the type? e.g.
```
type AllowedStrings = 'one of' | 'two of' | 'three of';
// ....
public myProp: AllowedStrings = 'one of';
```
Etc?

Reply:
I don't get what you mean. What is if the type is "numeric" and max is 9999. Would you list all values?

Reply:
Seems like maybe this is both “state of the art” and a little early, but possibly the approach that would make sense. https://github.com/microsoft/TypeScript/issues/43505 Would have thought there’d already be types for this, but without the min/max makes more sense.

Reply:
What is if it's a number enum? as I said in #51 
The combined type 1 | 2 | 3 would not help much